app/lib/app/features/home/data/home_repository_impl.dart

```diff
@@ -1,12 +1,29 @@
+import 'package:coopartilhar/app/features/home/data/order_adapter.dart';
 import 'package:coopartilhar/app/features/home/interactor/home_interactor.dart';
 import 'package:coopartilhar/app/features/home/interactor/interfaces/i_home_repository.dart';
 import 'package:core_module/core_module.dart';
 import 'package:coopartilhar/app/features/home/interactor/order_entity.dart';
 
 class HomeRepositoryImpl implements IHomeRepository {
+  final IRestClient restClient;
+
+  const HomeRepositoryImpl(this.restClient);
+
   @override
-  Future<Output<List<OrderEntity>>> getOrders(CategoryHelp category) {
-    // TODO: implement getOrders
-    throw UnimplementedError();
+  Future<Output<List<OrderEntity>>> getOrders(TagType tagType) async {
+    try {
+      final response = await restClient.get(RestClientRequest(path: '/feed'));
+
+      final json = response.data as List;
+
+      final ordersFilter = json.where((e) => e.tag == tagType.name);
```

Comment:
```suggestion
      final ordersFilter = json.firstWhereOrNull((e) => e.tag == tagType.name);
```
 para evitar quebrar caso não venha os valores esperados da requisição.

Reply:
precisa importar [collection.dart](https://api.flutter.dev/flutter/package-collection_collection/package-collection_collection-library.html)